app/assets/v2/js/user_card.js

```diff
@@ -100,7 +107,15 @@ const renderPopOverData = function(data) {
     {'type': 'Grants', 'sent': 'Fund', 'received': 'Contrib'}
   );
 
-  let mount_graph = [ tips_total_percent, bounties_total_percent, grants_total_percent ];
+  let mount_graph;
+
+  if (data.profile.has_ptoken) {
+    mount_graph = [ tips_total_percent, token_total_percent, bounties_total_percent, grants_total_percent ];
+  } else {
+    mount_graph = [ tips_total_percent, bounties_total_percent, grants_total_percent ];
+  }
```

Comment:
maybe something more cleaner like 
`if (data.profile.has_ptoken) {
    mount_graph.push(token_total_percent)
}`

Reply:
I'll defer to @zoek1 on this one

Reply:
done